examples/Raytracer_Benchmark/launcher.cu

```diff
@@ -0,0 +1,13 @@
+// SPDX-FileCopyrightText: 2020 CERN
+// SPDX-License-Identifier: Apache-2.0
+
+#include "RaytraceBenchmark.hpp"
+
+
+int executePipelineGPU(const vecgeom::cxx::VPlacedVolume *world, int argc, char *argv[])
+{
+  int result;
+  result = runSimulation<copcore::BackendType::CUDA>(world, argc, argv);
+  return result;
+}
+
```

Comment:
This can sit in RaytraceBenchmarkGPU file, no?

Reply:
Right